addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureFileInclusionScanRule.java

```diff
@@ -490,28 +452,21 @@ public void scan(HttpMessage originalmsg, String paramname, String paramvalue) {
                             // parameters on the same request (to reduce performance impact)
                             return;
                         } else {
-                            if (log.isDebugEnabled()) {
-                                log.debug(
-                                        "The output for the WAR/EAR code filename ["
-                                                + prefixedUrlfilename
-                                                + "] does not sufficiently differ in length from that of the random parameter, at "
-                                                + randomversussourcefilenamematchpercentage
-                                                + "%, compared to a threshold of "
-                                                + this.thresholdPercentage
-                                                + "%");
-                            }
+                            log.debug(
+                                    "The output for the WAR/EAR code filename [{}] does not sufficiently differ in length from that of the random parameter, at {}%, compared to a threshold of {}%",
+                                    prefixedUrlfilename,
+                                    randomversussourcefilenamematchpercentage,
+                                    this.thresholdPercentage);
                         }
                         if (this.isStop()) {
-                            if (log.isDebugEnabled()) log.debug("Stopped, due to a user request");
+                            log.debug("Stopped, due to a user request");
                             return;
                         }
                     }
                 }
             } else {
-                if (log.isDebugEnabled()) {
-                    log.debug(
-                            "Not checking for EAR/WAR files for this request, since the Attack Strength is not HIGH or INSANE");
-                }
+                log.debug(
```

Comment:
Adding comment here since the target line is not visible, the following `error` needs `{}`.